skepticoin/datatypes.py

```diff
@@ -125,6 +125,11 @@ def stream_serialize(self, f: BinaryIO) -> None:
 
 
 class Transaction(Serializable):
+
+    version: int
+    inputs: List[Input]
+    outputs: List[Output]
+
```

Comment:
These would be type annotations for fields of the class Transaction, not instances of it.

Fields of class instances usually get the right type (from mypy) in `__init__()`, either from default constants by assigning  from typed arguments. 

Please remove these lines.

Reply:
fixed @ f594977